.changeset/loud-insects-camp.md

```diff
@@ -1,5 +1,5 @@
 ---
-"stylelint": major
+"stylelint": minor
 ---
 
-Changed: `.js` extension to `.mjs` and `.cjs`
+Refactored: `.js` extension to `.mjs` and `.cjs`
```

Comment:
[ask] Just to confirm, should we also update the maintainer guide to add `Refactored`? Or can we consider `Refactored` as just an exception?

https://github.com/stylelint/stylelint/blob/a83af3eb88551b524a5bccd0f2bb3adbcaf9c053/docs/maintainer-guide/pull-requests.md?plain=1#L23

Reply:
`refactor` is listed in the [README](https://github.com/conventional-changelog/conventional-changelog-config-spec/blob/master/versions/2.2.0/README.md#types) and the [schema](https://github.com/conventional-changelog/conventional-changelog-config-spec/blob/238093090c14bd7d5151eb5316e635623ce633f9/versions/2.2.0/schema.json#L24).